libraries/AP_GyroFFT/AP_GyroFFT.cpp

```diff
@@ -174,6 +200,14 @@ void AP_GyroFFT::init(uint32_t target_looptime_us)
 #else
     _window_size = constrain_int16(_window_size, 32, 256);
 #endif
+    // number of samples needed before a new frame can be processed
+    _window_overlap = constrain_float(_window_overlap, 0.0f, 0.9f);
+    _window_overlap.save();
```

Comment:
why force save() ? Maybe use set_and_save_ifchanged() ? That would avoid the traffic to the storage device if not needed
Or even better, don't save this at all. We generally try to avoid changing user set parameters

Reply:
ok, the user may have set an invalid value - this corrects that. I'll use set_and_save_ifchanged()

Reply:
Actually not saving it now